tests/baselines/reference/smartSelection_JSDocTags4.baseline

```diff
@@ -0,0 +1,43 @@
+/**
+ * @typedef {object} Foo
+ * @property {string} a
+ * @property {number} b
+ * @property {/**/number} c
```

Comment:
@andrewbranch What do we need to select in this case?
1. `number`
2. `@property {number} c`
3. ?